src/app/seeds/development/challenge-platforms.json

```diff
@@ -1,18 +1,53 @@
 {
   "challengePlatforms": [
     {
-      "id": "61393be2ebf3a87e774058d9",
+      "id": "613a2d2b3084347fbc8efc03",
```

Comment:
Why are you changing this Id?

Reply:
@tschaffter I was manually adding those platform and challenges data. I manually made platformId in challenges to match the id from seeding challengePlatform.

Not sure if it just happens on my end. If I used current seeding function, platformId in challenge does not match the id in the id in the challengePlatform.